sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/src/Generated/Models/UeInfo4G.cs

```diff
@@ -0,0 +1,189 @@
+        {
+            Imsi = imsi;
+            Imei = imei;
+            Imeisv = imeisv;
+            SessionInfo = sessionInfo;
+            PerUeTnla = perUeTnla;
+            MmeS1ApId = mmeS1ApId;
+            EnbS1ApId = enbS1ApId;
+            LastVisitedTai = lastVisitedTai;
+            UeState = ueState;
+            RrcEstablishmentCause = rrcEstablishmentCause;
+            UeUsageSetting = ueUsageSetting;
+            LastActivityOn = lastActivityOn;
+            NgeNbId = ngeNbId;
+            ENbId = eNbId;
+            N3IwfId = n3IwfId;
+            WagfId = wagfId;
+            TngfId = tngfId;
+            Nid = nid;
+            BitLength = bitLength;
+            GNBValue = gnbValue;
+            MccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc = mccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc;
+            MncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc = mncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc;
+            LocationType = locationType;
+            Tac = tac;
+            MccInfoConnectionInfoLocationInfoPlmnMcc = mccInfoConnectionInfoLocationInfoPlmnMcc;
+            MncInfoConnectionInfoLocationInfoPlmnMnc = mncInfoConnectionInfoLocationInfoPlmnMnc;
+            MTmsi = mTmsi;
+            GroupId = groupId;
+            Code = code;
+            MccInfoGutiPlmnMcc = mccInfoGutiPlmnMcc;
+            MncInfoGutiPlmnMnc = mncInfoGutiPlmnMnc;
+            RatType = ratType;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="UeInfo4G"/> for deserialization. </summary>
+        internal UeInfo4G()
+        {
+        }
+
+        /// <summary> International mobile subscriber identifier. </summary>
+        public string Imsi { get; set; }
+        /// <summary> International mobile equipment identity. </summary>
+        public string Imei { get; set; }
+        /// <summary> International mobile equipment identity – software version. </summary>
+        public string Imeisv { get; set; }
+        /// <summary> Gets the session info. </summary>
+        public IList<UeSessionInfo4G> SessionInfo { get; }
+        /// <summary> Per-UE transport network layer association. </summary>
+        public string PerUeTnla { get; set; }
+        /// <summary> MME S1AP identifier. </summary>
+        public int? MmeS1ApId { get; set; }
```

Comment:
Just want to double check the format of the ID

Reply:
same question for other ID properties in this class

Reply:
The format should be  uint. This generated using "dotnet build /t:GenerateCode".

Reply:
Refer https://github.com/Azure/azure-rest-api-specs/blob/main/specification/mobilenetwork/resource-manager/Microsoft.MobileNetwork/stable/2024-02-01/ueInfo.json